lib/console/deployments/pr/impl/bitbucket.ex

```diff
@@ -63,7 +63,85 @@ defmodule Console.Deployments.Pr.Impl.BitBucket do
     end
   end
 
-  def files(_, _), do: {:ok, []}
+  def files(conn, url) do
+    with {:ok, group, number} <- get_pull_id(url),
+         {:ok, conn} <- connection(conn) do
+      # First get PR details to get the title
+      case HTTPoison.get("#{conn.host}/repositories/#{URI.encode(group)}/pullrequests/#{number}", Connection.headers(conn)) do
+        {:ok, %HTTPoison.Response{status_code: 200, body: body}} ->
+          case Jason.decode(body) do
+            {:ok, pr} ->
+              # Then get the diff to get file changes
+              case HTTPoison.get("#{conn.host}/repositories/#{URI.encode(group)}/pullrequests/#{number}/diff", Connection.headers(conn)) do
+                {:ok, %HTTPoison.Response{status_code: 200, body: diff}} ->
+                  changes = parse_diff(diff)
+                  files = Enum.map(changes, fn change ->
+                    _file_status = cond do
```

Comment:
rm these (also usually a case is better than a cond, in this case it would be:

```elixir
case change do
%{"status" => "added"} -> "(added)"
...
end
```